sinks/cache/cache_impl.go

```diff
@@ -264,11 +306,13 @@ func (rc *realCache) GetEvents(start, end time.Time) []*Event {
 }
 
 func NewCache(bufferDuration time.Duration) Cache {
-	return &realCache{
+	rc := &realCache{
 		pods:           make(map[string]*podElement),
 		nodes:          make(map[string]*nodeElement),
 		events:         store.NewGCStore(store.NewTimeStore(), bufferDuration),
 		eventUIDs:      make(map[string]struct{}),
 		bufferDuration: bufferDuration,
 	}
+	go util.Until(rc.runGC, time.Second, util.NeverStop)
```

Comment:
Every second seems way too often to run GC that only garbage collects things on the order of minutes old. Can you increase the period between runs? Every minute seems like it'd be more than enough, since holding data an extra minute shouldn't pose much risk.

If you have this set so low for the sake of testing, then just make it configurable and set it lower in testing.